test/xslt/test_exception_handling.rb

```diff
@@ -28,7 +28,7 @@ def test_java_exception_handling
           xsl.transform xml
           fail('It should not get here')
         rescue RuntimeError => e
-          assert_match(/HIERARCHY_REQUEST_ERR/, e.to_s, 'The exception message does not contain the expected information')
+          assert_match(/Can't have more than one root/, e.to_s, 'The exception message does not contain the expected information')
         end
```

Comment:
It looks like the exception message has changed or may be the different implementations are throwing different exceptions. Either case, this change looks good to me and is more readable.